README.md

```diff
@@ -172,6 +172,23 @@ Awspec.configure do |config|
 end
 ```
 
+### Advanced Tips: Setting A Custom Endpoint And Testing Locally
+
+Set the `aws_custom_endpoint` environment variable to tell awspec to use a different endpoint to connect to aws:
+
+```
+aws_custom_endpoint=http://localhost:5000 bundle exec rake spec
+```
+
+This can be used with [Moto's Standalone Server Mode](http://docs.getmoto.org/en/latest/docs/server_mode.html) to run all your aws tests locally. For example, to create resources on the local server using the aws cli, run:
```

Comment:
Moto is not the only tool that emulates AWS services locally.
I don't think an introduction to specific tools is necessary.

Reply:
That makes sense! I rewrote this section to be more generic, because you reminded me that local testing is not the only use case for this.

Reply:
Thank you!!!!